swingset-demo/src/main/java/com/nqadmin/swingset/demo/Example3.java

```diff
@@ -186,7 +186,8 @@ public void performPreInsertOps() {
 					rs.close();
 
 				// SET OTHER DEFAULTS
-//					 cmbSupplierName.setSelectedValue(0);
+					 logger.debug("Setting default for Supplier Name mapping to 2.");
+					 cmbSupplierName.setSelectedMapping((long) 2);
```

Comment:
This was just to have a default of something other than null to make sure it was being picked up properly for an insert.